### PR TITLE
EZP-29444: Create starter steps doc page

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://jira.ez.no/browse/EZP-29444
| Versions      | all, with modifications

Adds a section about first common steps to take after installing clean eZ Platform.

**Note**: changes will probably be needed for 1.x versions, maybe in a separate PR.

### CODE REVIEW

File: docs/getting_started/first_steps.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+    - [Setting up multi-language SiteAccesses and corresponding translations](../cookbook/setting_up_multi_language_siteaccesses.md)
+
+## Add a design
+
+The design engine enables you to use different themes consisting of templates and assets.
+Each theme is stored in a separate folder and assigned to a SiteAccess.
+
+To create a new theme:
+
+1\. Add the following configuration at the bottom of `app/config/ezplatform.yml` (at the same level at `ezpublish`):
```
Comment (MagdalenaZuba) [inline docs/getting_started/first_steps.md]: at the same level as

File: docs/getting_started/first_steps.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+4\. Go back to the `content_view` configuration for the blog post. Change the path to the template so that it points to the folder for the correct design:
+`template: '@ezdesign\full\blog_post.html.twig'`
+
+This means that the app will look for the `blog_post.html.twig` file in a folder relevant for the SiteAccess: `de_design` for the `de` SiteAccess, or `site_design` for other SiteAccesses in `site_group`.
+
+5\. Create a `themes` folder under `app/Resources/views`, and two folders under it: `de_design` and `site_design`.
+
+6\. Move the existing `full\blog_post.html.twig` file under `site_design`.
+
+7\. Copy it also under `de_design`. Modify the second one, so you can preview the effect.
```
Comment (MagdalenaZuba) [inline docs/getting_started/first_steps.md]: Modify the second one - how?

File: docs/getting_started/first_steps.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+
+7\. Save the Content Type.
+
+!!! tip "More information"
+
+    - [Content model](../guide/content_model.md)
+
+## Create Twig templates and match then with view config
+
+To display Content in the front page you need to define content views and templates.
```
[the remark this answers is not in the document]
Comment (DominikaK) [reply docs/getting_started/first_steps.md]: We use "content view" without capitalization in https://doc.ezplatform.com/en/2.2/guide/content_rendering/

[the remark this answers is not in the document]
Comment (MagdalenaZuba) [reply docs/getting_started/first_steps.md]: ok :)

**Pull request merged**